src/clr-addons/quick-list/quick-list.component.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'clr-quick-list',
+  templateUrl: './quick-list.component.html',
+  styleUrls: ['./quick-list.component.scss'],
```

Comment:
styles should not be inserted directly to components, see e.g. combobox how this was done (also add the @include).
Hint: you need to add it to components.clr-addons.scss instead.